frontend/src/features/admin-form/settings/queries.ts

```diff
@@ -31,6 +38,26 @@ export const useAdminFormSettings = (): UseQueryResult<FormSettings> => {
   )
 }
 
+/**
+ * Need to use React query's `refetch` to trigger a fetch.
+ * @precondition Must be wrapped in a Router as `useParam` is used.
```

Comment:
Typo here? Doesn't look like we are using `useParam` in the function body.

Reply:
thanks for the catch, i changed the useQuery logic to use `fetchQuery` but forgot to update the comment!